doc/source/examples/dem/small-scale-rotating-drum/small-scale-rotating-drum.rst

```diff
@@ -0,0 +1,187 @@
+    	set insertion frequency            		 	= 100000
+    	set insertion box minimum x            	 	= -0.05
+    	set insertion box minimum y            	        = 0.
+    	set insertion box minimum z            	        = -0.04
+    	set insertion box maximum x            	        = 0.05
+    	set insertion box maximum y           	 	= 0.04
+    	set insertion box maximum z            	        = 0.04
+    	set insertion distance threshold			= 1.1
+    	set insertion random number range			= 0.05
+    	set insertion random number seed			= 19
+    end
+
+Restart files are written once the packing ends. The restart files are used to start the DEM simulation with the imposed rotating boundary condition.
+
+Lagrangian physical properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The particles are mono-dispersed with a radius of 0.0015 m and a density of 2500 kg/m3, respectively. All other particles' physical parameters are taken arbitrary and should be changed based on the physical properties and the experimental values obtained through calibration.
+
+.. code-block:: text
+
+    subsection lagrangian physical properties
+        set gx            		 						= 0.0
+        set gy            		 						= -9.81
+        set gz            		 						= 0.0
+        set number of particle types	                = 1
+            subsection particle type 0
+            set size distribution type					= uniform
+                set diameter            	 			= 0.003
+                set number              				= 20000
+                set density particles  	 				= 2500
+                set young modulus particles         	= 100000000
+                set poisson ratio particles          	= 0.24
+                set restitution coefficient particles	= 0.97
+                set friction coefficient particles      = 0.3
+                set rolling friction particles         	= 0.01
+        end
+        set young modulus wall            				= 100000000
+        set poisson ratio wall            				= 0.24
+        set restitution coefficient wall           		= 0.85
+        set friction coefficient wall         			= 0.35
+        set rolling friction wall         	      	  	= 0.01
+    end
+
+
+Model parameters
+~~~~~~~~~~~~~~~~~
+
+The load balancing method used is the dynamic method. This method checks frequently if load balancing should be applied based on a user inputted frequency. Load balancing is dynamically applied if a certain condition is applied. More details regarding load balancing are explained in the `Rotating Drum example <../rotating-drum/rotating-drum.html>`_. 
+
+.. code-block:: text
+
+    subsection model parameters
+      set contact detection method 		   	 = dynamic
+      set dynamic contact search size coefficient	 = 0.8
+      set neighborhood threshold			 = 1.3
+      set load balance method				 = dynamic
+      set load balance threshold			 = 0.5
+      set dynamic load balance check frequency		 = 100000
```

Comment:
You can put a lower frequency than that. 100k is not frequent enough. on my machine I tried with 10k and it was working well.